db/seeds/development/sites.seeds.rb

```diff
@@ -1,6 +1,7 @@
 Site.create!(
   name: "site1",
   fqdn: ENV["HEROKU_APP_NAME"].present? ? "#{ENV['HEROKU_APP_NAME']}.herokuapp.com" : "localhost",
+  category_title_format: "title (%{name})",
```

Comment:
This seems a optional value.
So can you consider using factory_girls' trait if necessary?